src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMHead.java

```diff
@@ -36,12 +36,31 @@
 
     private static final long serialVersionUID = 1;
 
-    public PullRequestSCMHead(int number) {
+    private String user;
```

Comment:
:ant: Here we are representing the login, not a `user` entity. In fact, in the PR context, the right concept is `assignee`.